File: BE/Negocio/EvaluaciónProveedorBE.vb
Public Class EvaluaciónProveedorBE

    Inherits ID

    Public Property Proveedor As BE.ProveedorBE
    Public Property CalificacionTiempos As Integer
    Public Property CalificacionAtencion As Integer
    Public Property CalificacionComunicacion As Integer
    Public Property CalificacionCalidad As Integer
    Public Property Cantidad As Integer
    Public Property Fecha As String


    Public Function Promedio() As Double
        Dim prom = ((CalificacionTiempos + CalificacionComunicacion + CalificacionCalidad + CalificacionAtencion) / 4)
        Return prom
    End Function

    Sub New()

    End Sub

End Class

File: BE_2/Negocio/EvaluaciónProveedorBE.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Security;
using System.Text;
using System.Threading.Tasks;
using Microsoft.VisualBasic;

public class EvaluaciónProveedorBE : ID
{
    public ProveedorBE Proveedor { get; set; }
    public int CalificacionTiempos { get; set; }
    public int CalificacionAtencion { get; set; }
    public int CalificacionComunicacion { get; set; }
    public int CalificacionCalidad { get; set; }
    public int Cantidad { get; set; }
    public string Fecha { get; set; }


    public double Promedio()
    {
        var prom = ((CalificacionTiempos + CalificacionComunicacion + CalificacionCalidad + CalificacionAtencion) / (double)4);
        return prom;
    }
}

